.github/workflows/cd.yml

```diff
@@ -7,31 +7,51 @@ on:
   workflow_dispatch:
     inputs:
       environement:
-        description: "Environment (dev/test)"
+        description: "Environment (dev/test/prod)"
         required: true
         default: "dev"
 
 jobs:
+  push:
+    runs-on: ubuntu-20.04
+
+    if: github.event_name == 'push' && github.repository == 'bcgov/business-create-ui'
+    environment:
+      name: "dev"
```

Comment:
So how does this work? Does it always do a push, which deploys "main branch" to dev environment?

Is it possible to simply "dispatch" dev -> test or test -> prod?

Reply:
Push commit can be only deployed to dev. Test and prod requires dispatch action to deploy. Also need reviewers to approve it. I may explain it in Tech review next time.

Reply:
Sounds good.